doc/devel/testing.rst

```diff
@@ -144,13 +144,14 @@ tests it::
        fig, ax = plt.subplots()
        ax.plot(range(10), linestyle=(0, (3, 3)), lw=5)
 
-The first time this test is run, there will be no baseline image to compare
-against, so the test will fail.  Copy the output images (in this case
-:file:`result_images/test_lines/test_line_dashes.png`) to the correct
-subdirectory of :file:`baseline_images` tree in the source directory (in this
-case :file:`lib/matplotlib/tests/baseline_images/test_lines`).  Put this new
-file under source code revision control (with ``git add``).  When rerunning
-the tests, they should now pass.
+
+What should a developer do if they want to change any baseline images?
+----------------------------------------------------------------------
+In case any test doing comparison against baseline images is changed, then the baseline image was changed in the earlier
```

Comment:
Maybe reduce this to "If you are missing baseline images run with `--generate-images`, see [link] for more details"?

Something I do with my own writing is after I write a sentence I flip the order (put the second half first).  On one hand it makes sense to tell the reader the important information first, but readers usually do better being told why they should care about finishing the sentence rather than trying to guess why what you just told them was important.  For example

> In the previous flow the committed baseline images were changed when the tests generating them changed.

reads better than

> In case any test doing comparison against baseline images is changed, then the baseline image was changed in the earlier flow.

(and is shorter to boot!)